tests/testdata/leanimports.py

```diff
@@ -4,11 +4,18 @@
 import sys
 
 # block slow imports
-for unwanted in ["deepspeed", "llama_cpp", "torch", "transformers", "vllm"]:
+for unwanted in ["deepspeed", "llama_cpp", "torch", "vllm"]:
```

Comment:
Why are we removing transformers from this? It was in here on purpose, as we don't want anything eagerly loading transformers. I'm also confused by the docling_core import below here, as that's seems inappropriate for what this tests.

Reply:
Fair point @bbrowning and we had to remove transformers here because `docling-core[chunking]` had dependencies for `transformers >=4.34.0` and `semchunk >= 2.2.0`. Having `transformers` in the slow imports was not letting us use docling-core[chunking] that is essential for hybrid chunking. Would appreciate if you have alternate suggestions of handling this. 

And as far as the `docling-core` import is concerned, I am going to take that away. @courtneypacheco and I were testing if it throws out an error there.

Reply:
So, the purpose of this test is just to ensure we're not doing top-level imports of these heavyweight libraries that will slow down things like the InstructLab or SDG CLIs. However, that doesn't mean you can't have dependencies that use these. It just means your import statements for code that triggers a transformer import needs to not be at the top level of a module and instead move the import statement closer to where that code is used, typically within some method call.